new.js

```diff
@@ -8,14 +8,26 @@ $('#entryForm').change(function() {
   newEntry.publishedOn = new Date();
   newEntry.body = marked($('#articleBody').val());
 
-  var previewTemplateScript = $('#article-template').html();
-  var previewTemplate = Handlebars.compile(previewTemplateScript);
-  var previewHTML = previewTemplate(newEntry);
-
-  $('#preview').empty().append(previewHTML);
-  $('pre code').each(function (i, block){
-    hljs.highlightBlock(block);
-  });
+  // $.get('templates/article.handlebars', function(result){
+  //   var articleTemplateScript = result;
+  //   var articleTemplate = Handlebars.compile(articleTemplateScript);
+  //   var articleHTML = articleTemplate(articles);
+  //   $('.articles').append(articleHTML);
+
+  $.get('templates/article.handlebars', function(result){
+      var previewTemplateScript = $('#article-template').html();
+      var previewTemplate = Handlebars.compile(previewTemplateScript);
+      var previewHTML = previewTemplate(newEntry);
+      $('#preview').empty().append(previewHTML);
+      $('pre code').each(function (i, block){
+        hljs.highlightBlock(block);
+      }
+    });
```

Comment:
This is giving me an error: `Parsing Error Unexpected Token }`. This usually means braces or parenthesis are unbalanced.